src/components/App.js

```diff
@@ -1,27 +1,13 @@
-import logo from './logo.svg';
-import './App.css';
+import '../styles/App.css';
+import React from 'react';
+import ButtonPanel from './ButtonPanel';
+import Display from './Display';
 
 function App() {
   return (
     <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit
-          {' '}
-          <code>src/App.js</code>
-          {' '}
-          and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
```

Comment:
- [ ] Kindly wrap the Display and ButtonPanel components in [React.Fragment](https://reactjs.org/docs/fragments.html#short-syntax) short syntax instead ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone2/app/ReactFragment.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/44896092/))

Reply:
Fixed